litmus-portal/frontend/src/components/YamlEditor/styles.ts

```diff
@@ -40,87 +58,268 @@ const useStyles = makeStyles((theme: Theme) => ({
 
   editorGrid: {
     overflow: 'auto',
-    maxHeight: '34.5rem',
-    width: '92%',
+    height: '50vh',
+    width: '100%',
+    '&::-webkit-scrollbar': {
+      width: '0.2em',
+    },
+    '&::-webkit-scrollbar-track': {
+      webkitBoxShadow: 'inset 0 0 6px rgba(0,0,0,0.00)',
```

Comment:
why an rgba of 0.00???
use theme color here...if it doesn't need an opacity use `common.black` color